package.json

```diff
@@ -7,6 +7,7 @@
     "build": "next build",
     "start": "next start",
     "eslint": "eslint .",
+    "stylelint-check": "stylelint-config-prettier-check",
```

Comment:
The stylelint script to check the css files seems to be missing.

Reply:
npm run stylelint-check? I can use that and get the following reply: > fullstack-template@0.1.0 stylelint-check
> stylelint-config-prettier-check

No conflicting rules detected in your stylelint configuration!